### ISSUE TITLE
App UX challenge: how to break the bad news?

### ISSUE THREAD

**Issue by kugelfish42** [DP-3T/documents#186]
If the app detects one of more exposure events how should it notify the user, how much details should it provide and what hopefully relevant and actionably recommendations should it make.

How could an app acquire some level of good bedside manners when having to break to the user the bad news of potentially having been exposed to a potentially harmful and live threatening disease?

How much information should be shared with the user? On one hand, showing the times and durations for each relevant contact event would could help to diagnose and dismiss likely false positives, which also requires some knowledge about the inner workings of the app and Bluetooth radio characteristics. Showing these details might also allow users to identify who the contact might be and might lead to some uncontrolled aggression towards that person.

This is a communications challenge that should probably not be attempted to be solved alone by engineers and tech-affine people.

**Comment by kaepora** [comment]
> How could an app acquire some level of good bedside manners when having to break to the user the bad news of potentially having been exposed to a potentially harmful and live threatening disease?

The hard truth is that it can't. There's simply no way an app can acquire appropriate "bedside manners" for such situations, and so my recommendation would be for the app to deal with the issue the same way doctors have for centuries. Communicate the news objectively, calmly and simply.

In order to communicate my point accurately, I've designed two mockups:

**User receives a to-the-point notification immediately after the contact event is confirmed:**

<img width="300" alt="Screen Shot 2020-04-16 at 4 11 34 PM" src="https://user-images.githubusercontent.com/9953/79467290-eb56a900-7ffd-11ea-92ab-23d4d229b922.png">

**Upon opening the notification, user is presented with a page that allows them to:**

1. View more information about the date and location of the contact event (red "View button" below),
2. Contact their doctor/local health authorities,
3. Obtain advice on confinement practices.

<img width="300" alt="Screen Shot 2020-04-16 at 4 17 02 PM" src="https://user-images.githubusercontent.com/9953/79467307-f0b3f380-7ffd-11ea-974e-70bd84706253.png">

I am of the strong opinion that attempting to sugar-coat or, worse, delay the notification will backfire, leading to additional anxiety in the former and additional potential for spreading the infection in the latter. This is an issue that deserves to be discussed at length, but right now I really think this is the only way to go here.

**Comment by kaepora** [comment]
@cascremers A wise choice.

**Comment by adewes** [comment]
Personally I think you should give this some consideration within the project and involve both UX/UI experts and possibly even psychologists as well as medical professionals. As the false positive rate with the BLE approach can potentially be very high and as a single person can potentially generate thousands of contacts over the infection period I think it's crucial to provide additional context to the user at risk by e.g. specifying when the contact occurred and how "strong" it was. Otherwise there's a large risk that people will ignore the notification. From my experience this can occur really quickly if a system produces too many false positives or fails to provide relevant context that enables users to investigate and verify the claims (even if only to allow the user to rationalize the notification). Providing as much context as possible is therefore crucial in my opinion. Also, as a single infected app user can theoretically send hundreds of people into self-quarantine, many of which maybe didn't have a relevant, risky contact and would be able to ascertain that if given enough context e.g. about the exact time the contact supposedly happened.

Building a secure and (mostly) privacy-preserving protocol  is one important part of the app, but you should not forget the psychological and user experience aspects, as those will be critical for the acceptance of the app as well. Leaving their implementation open to the individual providers/countries is not a good strategy in my opinion.

**Comment by kugelfish42** [comment]
Agreed that this might be off-topic for the crypto/privacy/security/protocol-design centric discussions here, but it would be important for the success of the effort. Is there a similar forum where app developers working for respective authorities could connect for sharing strategies and best practices and tap into the collective expertise of the UXR, UXD or policy & communications communities?

Once finished, these apps will als need to be installed and used by a large number of users - they will need to go viral faster and more persuasively than any app before. This will require a wide body of trusted experts, leaders and influencers to stand behind the effort and recommending its use, but also a lot of old-fashioned PSA marketing campaigns, this time focused on app installs.

**Comment by henriterhofte** [comment]
Should the app be designed in such a way that a user can plausibly deny having received an at risk notification? If you can show your at-risk-notification-status, and don't have plausible denial, you also be forced to show your at-risk notification status. This might be an issue related to both privacy and ethics.

**Comment by heroichornet** [comment]
The optimal behaviour of a potential invected person differs depending on the phase of the pandemic we are in. Therefore the app should also encourage different things. If we are in a lock down then you should be encouraged to self isolate as if you have been infected. You should contact medical professionals only by phone first and then get tested.

If however we are in "test and trace" phase the app should not only encourage strict isolation but also provide the person with the optimal place and time to get tested. The person can contact a hotline with a code and this will provide them with a drive through test date and time.

**Comment by christiano-git** [comment]
I would like to be informed with facts that allow to see a upside and what I can do.

E.g
- Most infection cure with mild or no symptoms.
- You can contribute to lower the infections. - Isolate yourself to protect the society.
Contact your doctor or the hotline.

**Comment by ChAMeier** [comment]
Adding/replying to the comment of kugelfish42 "**How much information should be shared with the user?**"

I am a strong proponent of using technology to fight such a pandemic. And I am willing to put myself in isolation to prevent spreading of the virus and to protect others.

However, I am also a strong believer that **the tracing app will produce a lot of false positives**. With a simple notification "you may have been in contact..." and without any further information a user is not able to verify the likelihood of an infection and to identify false positives such as:
- I left my phone at the coffee machine, somebody brought it to the reception desk where I picked it up 2 hours later; during that time the phone was close to an infected person
- I had lunch on my own in a cafeteria; there are glass walls between the boots; I was sitting back-to-back to a person unknown to me; our phones were close and connected.  

I propose that the **notification is supplemented with a timestamp** of the event of contact so that the user is able to identify false positives. Otherwise an app user may not step forward and put him-/herself and others in an unjustified isolation. Thus, there is the **risk that the effectiveness of the tracing app is reduced**.
